subsys/bluetooth/controller/ll_sw/ull_llcp_common.c

```diff
@@ -843,36 +844,25 @@ static void lp_comm_st_wait_rx(struct ll_conn *conn, struct proc_ctx *ctx, uint8
 	}
 }
 
-static void lp_comm_st_wait_ntf(struct ll_conn *conn, struct proc_ctx *ctx, uint8_t evt,
+static void lp_comm_st_wait_ntf_avail(struct ll_conn *conn, struct proc_ctx *ctx, uint8_t evt,
```

Comment:
When transitioning to LP_COMMON_STATE_WAIT_NTF_AVAIL the procedure TO is reset, but while inside the state itself (as long as no NTF is available), the TO is not reset is that on purpose?